app/views/users/dashboard/index.html.erb

```diff
@@ -4,6 +4,7 @@
 <p>Following: <%= @user.user_info[:followingCount] %></p>
 <p>Followers: <%= @user.user_info[:followersCount] %></p>
 <p>My Scores:</p>
+<%= link_to 'Create Score', '/scores/new' %>
```

Comment:
just put this link here for now, could move it wherever. My thinking on routing it to an actual page at first was one, I don't know javascript to make that dynamic and two I need a title from a user so a form seemed to be the simplest solution. I think a more dynamic javascript solution would be cooler but baby steps